src/frontend/context/fagsak/FagsakContext.tsx

```diff
@@ -126,7 +107,7 @@ export const FagsakProvider = (props: PropsWithChildren) => {
     };
 
     const oppdaterKlagebehandlingerPåFagsak = () => {
-        const fagsakId = hentDataFraRessurs(minimalFagsakRessurs)?.id;
+        const fagsakId = fagsak?.id;
```

Comment:
Her vet man vel at fagsak er definert

```suggestion
        const fagsakId = fagsak.id;
```

Du kan og vurdere å skrive om til noe ala dette:

```
const oppdaterKlagebehandlingerPåFagsak = () => {
        request<void, IKlagebehandling[]>({
            method: 'GET',
            url: `/familie-ks-sak/api/fagsaker/${fagsak.id}/hent-klagebehandlinger`,
            påvirkerSystemLaster: true,
        }).then(klagebehandlingerRessurs => settKlagebehandlinger(klagebehandlingerRessurs));
    };
```